ui_components/src/travel/Map.js

```diff
@@ -186,11 +192,47 @@ function MapLocations({ locations, tileWidth, tileHeight }) {
                     <div className='map_locations_tooltip'>{location.name}</div>
                     {location.objective_image &&
                         <div className='map_location_objective' style={{
-                            backgroundImage: "url(." + location.objective_image + ")",
+                        backgroundImage: "url(." + location.objective_image + ")",
+                        zIndex: 5,
                         }}></div>
                     }
                 </div>
             ))}
         </div>
     );
+}
+
+function ScoutLocations({ scoutData, tileWidth, tileHeight, playerId }) {
```

Comment:
Wdyt about a more descriptive name, something like this?
```suggestion
function MapNearbyPlayers({ scoutData, tileWidth, tileHeight, playerId }) {
```

Reply:
Fixed in commit.